src/tri/_tri.cpp

```diff
@@ -220,14 +218,37 @@ Triangulation::Triangulation(const CoordinateArray& x,
                              const MaskArray& mask,
                              const EdgeArray& edges,
                              const NeighborArray& neighbors,
-                             int correct_triangle_orientations)
+                             bool correct_triangle_orientations)
     : _x(x),
       _y(y),
       _triangles(triangles),
       _mask(mask),
       _edges(edges),
       _neighbors(neighbors)
 {
+    if (_x.ndim() != 1 || _y.ndim() != 1 || _x.shape(0) != _y.shape(0))
```

Comment:
Argument type checking is handled automatically by `pybind11`. Here we need to check that array shapes are consistent. These checks were originally in the wrapper functions that no longer exist.